### PR TITLE
[JA] Add Beatmap related pages and resolve some pages' errors

### PR DESCRIPTION
**Pull request by JackTriton0901**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

## Changes

Add `ja.md` for :
- `Beatmap/Beatmap collaborations`
- `Beatmap/Beatmap host`
- `Beatmap/Genre and Language`
- `Beatmap/Guest difficulty`
- `Beatmap/Hype`
- `Beatmap/Marathon`
- `Game mode/osu!taiko`
- `Modding/Kudosu`
- `Modding/Mod for mod`
- `Modding/Mod request`
- `Modding/Modder`
- `Modding/Normal mod`
- `Modding/Timestamp`
- `Ranking Criteria/Difficulty Naming`
- `Ranking Criteria/Scaling BPM`
- `Skinning/Skinner`
- `Skinning/osu!taiko`

Delete duplicated information for `Bit rate` 

Resolve markdown error of `Skinning`
______
Where can I find Japanese reviewer

- @TicClick: add `SKIP_WIKILINK_CHECK`

### CODE REVIEW

**Comment by JackTriton0901** [comment]
Fixed some files

**Comment by JackTriton0901** [comment]
Changed all `\r\n` to `\n` for `Modding/Kudosu`.

**Comment by JackTriton0901** [comment]
Checked on github.dev, `Kudosu` page’s `\r\n` is replaced to `\n`.

**Comment by Walavouchey** [comment]
in the future, please make separate pull requests for related pages. if you intend on getting this reviewed, then grouping everything into one pr will only make it take a while.

don't worry about the line ending errors for now - they can be fixed before merging.

**Comment by TicClick** [comment]
this definitely needs splitting into several small, at least thematic if not per-article pull requests. one more thing, please don't merge the `master` branch into yours, a maintainer will do that when necessary

**Comment by JackTriton0901** [comment]
To fix the link error on `Game mode/osu!taiko`, I have to edit `Gameplay/Grade`.
I'll do this on different pull request.

**Review by peppy — DISMISSED** [review]

**Comment by peppy** [comment]
I've done a prelim check of the translation and it's fine.

File: wiki/Beatmap/Beatmap_host/ja.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+stub: true
+---
+
+# ビートマップホスト
+
+**ビートマップホスト** は[ビートマップ](/wiki/Beatmap)のマネージャーであり投稿者のことである。ビートマップのほとんどの難易度を作成し[ゲスト難易度](/wiki/Beatmap/Guest_difficulty)の管理も行う。[beatmap ranking procedure](/wiki/Beatmap_ranking_procedure)を進行させるために[Mod](/wiki/Modding)してくれる人や[ビートマップノミネーター](/wiki/People/The_Team/Beatmap_Nominators) に依頼する役割もあります。
```
Comment (TicClick) [inline wiki/Beatmap/Beatmap_host/ja.md:7]: "beatmap ranking procedure" needs to be translated -- it's a regular phrase, not a term

File: wiki/Beatmap/Genre_and_language/ja.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# ジャンルと言語
+
+![A screenshot of genre, language, and tags on a beatmap page](img/beatmap-info.png "ビートマップページに掲載されているジャンルと言語")
```
Comment (TicClick) [inline wiki/Beatmap/Genre_and_language/ja.md:10]: image tooltip needs to be translated

File: wiki/Beatmap/Genre_and_language/ja.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+サイト上に投稿された[ビートマップ](/wiki/Beatmap)には分類をしやすくするため**ジャンル**と**言語**の欄がある。これはビートマップの[メタデータ](/wiki/Beatmap_Editor/Song_Setup#song-and-map-metadata)の一つとして考えられる。
+
+ゲーム上に変化はないがosu!(lazer)ではそれによってジャンル分けができる。
+
+## ビートマップリスト
+
+![The available filters on the beatmap listing](img/beatmap-listing.png?1)
```
Comment (TicClick) [inline wiki/Beatmap/Genre_and_language/ja.md:18]: image tooltip needs to be translated

**Review by TicClick — CHANGES_REQUESTED** [review]
minor notes, fix this and I'll merge

File: wiki/Beatmap/Beatmap_host/ja.md (hunk 1 above)
Comment (JackTriton0901) [reply wiki/Beatmap/Beatmap_host/ja.md]: translated to ビートマップのランクをさせるために

File: wiki/Beatmap/Genre_and_language/ja.md (hunk 2 above)
Comment (JackTriton0901) [reply wiki/Beatmap/Genre_and_language/ja.md]: translated to ビートマップページに掲載されているジャンルと言語

File: wiki/Beatmap/Genre_and_language/ja.md (hunk 3 above)
Comment (JackTriton0901) [reply wiki/Beatmap/Genre_and_language/ja.md]: translated to ビートマップリストのフィルター

**Review by peppy — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**